src/elements/content-preview/ContentPreview.js

```diff
@@ -1227,8 +1269,18 @@ class ContentPreview extends React.PureComponent<Props, State> {
             });
         }
 
-        if (viewer) {
-            viewer.emit('scrolltoannotation', { id, target });
+        if (viewer && !deferScrollToOnload) {
+            viewer.emit(SCROLL_TO_ANNOTATION_EVENT, { id, target });
+        } else if (viewer && deferScrollToOnload) {
+            this.dynamicOnPreviewLoadAction = () => {
+                if (target?.location?.type === 'frame') {
+                    this.scrollToFrameAnnotation(id, target);
+                } else {
+                    const newViewer = this.getViewer();
+                    newViewer.emit(SCROLL_TO_ANNOTATION_EVENT, { id, target });
```

Comment:
Is it possible `newViewer` could be null here?

Reply:
Good point.